### PR TITLE
Integrate vanilla v1

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Swap out old theme and styling for latest Vanilla
- Clean out redundant JS files
- Tidy up indentation
- Convert header and footer
- Convert markup on homepage

## QA

- Pull code
- Sanity check file deletions
- Run `./run serve --watch`
- View homepage on http://0.0.0.0:8010/

Fixes https://github.com/ubuntudesign/vanilla-squad/issues/73,
Fixes https://github.com/ubuntudesign/vanilla-squad/issues/62

### CODE REVIEW

File: static/sass/styles.scss [hunk 1]
```diff
@@ -1,151 +1,11 @@
...
+$color-link: $color-brand;
 
-  	CONTENTS:
-	-----------------------------------------------------------------
-	@section general
-	@section main nav
-	@section footer
-	@section UA service-description
+// Import Vanilla Framework
+@import "../../node_modules/vanilla-framework/scss/build";
```
Comment (anthonydillon) [inline static/sass/styles.scss]: Your importing vanilla but not including it?

File: templates/cloud/index.html [hunk 2]
```diff
@@ -64,15 +64,15 @@
     </div>
     <div class="twelve-col vertical-divider">
         <div class="four-col">
```
Comment (anthonydillon) [inline templates/cloud/index.html]: Should you be using Vanilla grid here?

File: templates/takeovers/_hackathon.html [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
-    </div>
-    <div class="six-col last-col equal-height__item equal-height__align-vertically no-margin-bottom">
-      <div class="six-col last-col">
-        <h1 class="row-hero__heading">江苏·如皋</h1>
-        <p class="intro">万寿南路999号如皋软件园开源空间</p>
-        <p class="intro">2017年9月9至10日</p>
-        <p class="intro"><a class="link-cta-inverted" href="http://www.huodongxing.com/event/2398109794800?qd=cnsite" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'hackathon takeover', 'eventLabel' : 'Sign up now', 'eventValue' : undefined });">立刻报名</a></p>
+      </div>
+      <div class="col-6 u-vertically-center">
+        <div style="color: #fff;">
```
Comment (anthonydillon) [inline templates/takeovers/_hackathon.html]: Don't we have `card--overlay` for this situation?

File: templates/templates/base.html [hunk 4]
```diff
@@ -1,135 +1,76 @@
...
+  <link rel="shortcut icon" href="{{ STATIC_URL }}img/favicon.ico" type="image/x-icon" />
+  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="{{ STATIC_URL }}img/apple-touch-icon-144x144-precomposed.png">
+  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="{{ STATIC_URL }}img/apple-touch-icon-114x114-precomposed.png">
+  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="{{ STATIC_URL }}img/apple-touch-icon-72x72-precomposed.png">
+  <link rel="apple-touch-icon-precomposed" href="{{ STATIC_URL }}img/apple-touch-icon-precomposed.png">
+  <link type="text/plain" rel="author" href="{% versioned_static 'files/humans.txt' %}" />
+  <link rel="stylesheet" type="text/css" media="screen" href="{% versioned_static 'css/styles.css' %}" />
+  <!-- scripts -->
+  <script src="{{ STATIC_URL }}js/plugins/modernizr.2.7.1.js"></script>
+  <script src="{{ STATIC_URL }}js/plugins/Imager.js"></script>
```
Comment (anthonydillon) [inline templates/templates/base.html]: We can remove these two dependencies.

File: templates/templates/base.html [hunk 5]
```diff
@@ -1,135 +1,76 @@
...
+  {% include "templates/footer.html" %}
+
+  <!-- Use this only until Modernizr fixes Opera Mini background-size detection -->
+  <script>
+      var isOperaMini = (navigator.userAgent.indexOf('Opera Mini') > -1);
+      if (isOperaMini) {
         var root = document.documentElement;
         root.className += " opera-mini";
+      }
+  </script>
```
Comment (anthonydillon) [inline templates/templates/base.html]: This script is redundant.

File: templates/templates/base.html [hunk 6]
```diff
@@ -1,135 +1,76 @@
...
+  <script src="{{ STATIC_URL }}js/plugins/yui-combined.min.js"></script>
```
Comment (anthonydillon) [inline templates/templates/base.html]: We shouldn't support YUI any more.

File: templates/templates/header.html [hunk 7]
```diff
@@ -1,20 +1,29 @@
...
+        <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+        <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
+      </div>
+      <nav class="p-navigation__nav" role="menubar">
+        <span class="u-off-screen">
+          <a href="#main-content">Jump to main content</a>
+        </span>
+        <ul class="p-navigation__links" role="menu">
+          <li class="p-navigation__link" role="menuitem"><a{% if level_1 == 'internet-of-things' %} class="active"{% endif %} href="/internet-of-things">IoT</a></li>
+          <li class="p-navigation__link" role="menuitem"><a class="first{% if level_1 == 'desktop' %} active{% endif %}" href="/desktop">桌面OS</a></li>
```
Comment (anthonydillon) [inline templates/templates/header.html]: This the active class really `active`? Should it not be `is-active`?

File: templates/cloud/index.html (hunk 2 above)
Comment (barrymcgee) [reply templates/cloud/index.html]: I'll address this when I get to the /cloud section

File: static/sass/styles.scss (hunk 1 above)
Comment (barrymcgee) [reply static/sass/styles.scss]: `build.scss` imports and includes

screen shot 2017-09-01 at 15 41 44

Comment (anthonydillon) [reply static/sass/styles.scss]: I thought we imported and included vanilla in projects and `build.scss` was only used for building vanilla from within the vanilla project?

**Pull request merged**